src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,37 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_PAIRS_NUMBER = 10;
+    private final K[] keys = (K[]) new Object[MAX_PAIRS_NUMBER];
+    private final V[] values = (V[]) new Object[MAX_PAIRS_NUMBER];
+    private int size;
+
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < keys.length; i++) {
+            if (keys[i] == keys || ((keys[i] == key) || (keys[i] != null && keys[i].equals(key)))
+                    && values[i] != null) {
+                values[i] = value;
+                size--;
+            }
+        }
+        this.keys[size] = key;
+        this.values[size] = value;
```

Comment:
```suggestion
        keys[size] = key;
        values[size] = value;
```

we dont need this keyword here